problems/[230601] K Closest Points to Origin.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/k-closest-points-to-origin
+ *
+ * @param {number[][]} points
+ * @param {number} k
+ * @return {number[][]}
+ */
+const kClosest = function (points, k) {
+  const distance = [];
+
+  for (let i = 0; i < points.length; i += 1) {
+    distance.push(Math.sqrt(points[i][0] ** 2 + points[i][1] ** 2));
+  }
+
+  const indexArray = points
+    .map((_, index) => index)
+    .sort((a, b) => distance[a] - distance[b]);
+  const result = indexArray.map((index) => points[index]);
```

Comment:
```suggestion
  const indexArray = points
    .map((_, index) => index)
    .sort((a, b) => distance[a] - distance[b])
    .map((index) => points[index]);
```

sort() 메소드는 원본 배열을 변화시키는 mutable한 메소드인데, 정렬된 배열을 반환하기도 해서 chaining으로 연결할 수도 있어요~